test/box/broken_snapshot.test.lua

```diff
@@ -0,0 +1,123 @@
+
```

Comment:
As a rule we add gh-xxxx- prefix to test name

Reply:
ok